### PR TITLE
Pass specific type to SetWindowTextW

### LINKED ISSUE #17: Package stopped working in Windows
I noticed that my windows were not being renamed so I turned on debug to find the problem.
````
.......set_window_title.py", line 147, in rename_window_windows
    ctypes.windll.user32.SetWindowTextW(hwndSublime, new_title)
ctypes.ArgumentError: argument 2: <class 'TypeError'>: expected LP_c_ushort instance instead of str
````
I am using ST build 3207 on Win 10.0.17134
I have a simple fix that works for me but I don't know if it will work for everyone.
I will submit it as a PR.

### PR DESCRIPTION
**Pull request by BrennanGit**
Fixes #17 

This is a fix for an issue I was having after some time of this package working perfectly.

I'm not sure if anything changed in recent times and if my fix will work on older versions of ST or Windows but thought I'd leave it here for anyone else experiencing this issue.

### CODE REVIEW

File: set_window_title.py [hunk 1]
```diff
@@ -143,7 +144,9 @@ def rename_window_linux(self, window, official_title, new_title, debug=False):
   def rename_window_windows(self, new_title):
     # PX_WINDOW_CLASS is the ClassName of SublimeText, can be seen via a tool such as Nirsoft WinLister
     hwndSublime = ctypes.windll.user32.FindWindowA(b'PX_WINDOW_CLASS', None)
-    ctypes.windll.user32.SetWindowTextW(hwndSublime, new_title)
+    text = new_title.encode("utf16")	
+    buff = (ctypes.c_char * (len(text)+10))(*text)
```
Comment (ClaytonHughes) [inline set_window_title.py:148]: Thanks for the PR, I'm hoping this helps me, too.

Where does this 10 come from?

Comment (BrennanGit) [reply set_window_title.py:148]: I took it from the title setter method that was removed in the last commit. 
https://github.com/gwenzek/SublimeSetWindowTitle/commit/14e4d0bae8704d7b59e7856bba341de9cc86a94c#diff-a7908ba95124deb7948b618398904c6eL66

I imagine it was just there for a bit of breathing room.

**Comment by robertcollier4** [comment]
This shows the title wrong for me for Unicode filenames on on Win7, breaking Unicode foreign language titles, sometimes adding or replacing "?" or box characters. Other times I get a box prefixed to the title when the squiggly dirty marker is in the title. Is there a need to encode new_title to Unicode when it already should be Unicode since Sublime already handles strings in Unicode?

Some more testing would be advised on Windows 10 to see where it is breaking and why new_title is not already making a Unicode string. Do you have any forced ANSI flags setup in your Sublime python configuration to have it operating upon ansi strings instead of unicode strings? A double unicode encode might cause problems.

**Comment by gwenzek** [comment]
This is very weird.
Is this an API incompatibility between windows 7 and 10 ? that would be weird, Windows is generally very careful with this.

I'm not merging while other windows users have comfirmed it works for them.

**Comment by BrennanGit** [comment]
Hey, sorry I haven't followed up on this. It's been very busy at work and the fix works for me.

Happy to give any information you guys think would be helpful and to make any changes but deep investigation on multiple platforms is a bit more than I have time for right now.

Feel free to close if that's the best course of action for you.

**Comment by BrennanGit** [comment]
I'm going to close this PR as I'm using a different editor nowadays.

**Pull request closed without merging**